kep/61-real-load-aware-scheduling/61-real-load-aware-scheduling.md

```diff
@@ -87,7 +88,10 @@ The file will be stored in host file system, so it will be persisted across pod
 
 ### Scheduler Plugins
 
-This uses the scheduler framework of K8s to incorporate our custom real load aware scheduler plugins without modifying the core scheduler code.
+This uses the scheduler framework of K8s to incorporate our customized real load aware scheduler plugins without modifying the core scheduler code. The plugins we proposed mainly include the following two.
+
+- BestFitBinPack Plugin: It is a node sorting plugin that sorts nodes by their actual resource utilization in a way that all utilized nodes have around x% of utilization. We use x% = 50% below to describe the plugin.
+- Safe Balancing Plugin: It is a node sorting plugin that sorts nodes base on both the mean and the standard deviation of node resource utilization. It aims to balance not only the average load but also the risk caused by load variations.
```

Comment:
This is for discussion: https://github.com/kubernetes-sigs/scheduler-plugins/pull/61#discussion_r502605840